modyn/trainer_server/internal/dataset/data_utils.py

```diff
@@ -18,6 +18,7 @@ def prepare_dataloaders(
     storage_address: str,
     selector_address: str,
     training_id: int,
+    return_weights: bool = False,
```

Comment:
furthermore, I think this flag introduces many if/else in the OnlineDataset. If we have it (I am not sure yet whether we need it), I think it would rather implement this at the trainer server: the onlinedataset always returns weights, and the trainer server might choose to ignore them because that increases readability of the dataset by a lot i think